common/src/model_registration/register.py

```diff
@@ -17,11 +17,41 @@
 from azureml.core import Run
 from applicationinsights import TelemetryClient
 
-tc = TelemetryClient("71b954a8-6b7d-43f5-986c-3d3a6605d803")
 
 SUPPORTED_MODEL_ASSET_TYPES = [AssetTypes.CUSTOM_MODEL, AssetTypes.MLFLOW_MODEL]
 MLFLOW_MODEL_FOLDER = "mlflow_model_folder"
 
+tc = None
+
+
+def init_tc():
+    global tc
+    if tc is None:
+        try:
+            tc = TelemetryClient("71b954a8-6b7d-43f5-986c-3d3a6605d803")
+        except Exception as e:
+            print(f"Exception while initializing app insights: {e}")
+            tc = None
+
+
+def tc_log(message):
+    global tc
```

Comment:
Probably don't need this because you're not changing its value.

Reply:
added to avoid lint issue